hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/ResponseTemplate.java

```diff
@@ -162,9 +162,10 @@ void Exception(int errorCode, String className, @Nullable String message, StackT
     /**
      * @param readCount Number of items in the response.
```

Comment:
Bad description. This one was from before actually. Let's say you have an array of size 10. And you read with a filter and get items 0, 1, 3 and 7. The item list size is then 4 but the readCount will be 7. Since the read count isn't enough in the new functionalities, we send all indexes as well (and readCount is then not needed anymore and can be removed in a future version).